autonomous/autonomous.py

```diff
@@ -137,25 +144,68 @@ def move(self, tm):
             linearVelocityRef=linear_state.velocity * 0.15,  # used for feedforward
             angleRef=goal_rotation,
         )
+        self.chassis.drive_local(
+            self.chassis_speeds.vx, self.chassis_speeds.vy, self.chassis_speeds.omega
```

Comment:
Hmm. We should probably have a method to just set a `ChassisSpeeds` rather than unpacking and repacking it. Can be something for later.